### PR TITLE
Updated german translation

### PR DESCRIPTION
**Pull request by siegmund42**
Two things where a bit wrong, the others are additions.

### CODE REVIEW

File: ui/src/qlcplus_de_DE.ts [hunk 1]
```diff
@@ -1573,12 +1573,12 @@ Changes will be lost if you don&apos;t save them.</source>
     <message>
         <location filename="collectioneditor.ui" line="39"/>
         <source>Move the selected functions down</source>
-        <translation type="unfinished"></translation>
+        <translation>Ausgewählte Funktion nach unten schieben</translation>
```
Comment (kripton) [inline ui/src/qlcplus_de_DE.ts]: In english, it's plural

File: ui/src/qlcplus_de_DE.ts [hunk 2]
```diff
@@ -1573,12 +1573,12 @@ Changes will be lost if you don&apos;t save them.</source>
...
     </message>
     <message>
         <location filename="collectioneditor.ui" line="56"/>
         <source>Move the selected functions up</source>
-        <translation type="unfinished"></translation>
+        <translation>Ausgewählte Funktion nach oben schieben</translation>
```
Comment (kripton) [inline ui/src/qlcplus_de_DE.ts]: In english, it's plural

**Comment by kripton** [comment]
Thanks for updating them!
Looks good to me, except two singular vs. plural things I commented on inline.

**Comment by siegmund42** [comment]
You're right, but I tried to be consistent with this:
`        <translation>Ausgewählte Funktion entfernen</translation>` on line 1611.
Anyway, I'll change it to simply be a translation. If Massimo decides to adjust the English words I'll change it again.

**Comment by mcallegari** [comment]
thanks :+1:

**Pull request merged**